CMakeLists.txt

```diff
@@ -29,6 +29,13 @@ cmake_minimum_required(VERSION 2.6)
 project(gnuradio CXX C)
 enable_testing()
 
+# Apple only - look for libraries installed by macports
+if(APPLE)
+	SET(GCC_COVERAGE_COMPILE_FLAGS "-L/opt/local/lib -Qunused-arguments")
+	SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${GCC_COVERAGE_COMPILE_FLAGS}")
```

Comment:
I already tried to add the macports library path to the CMAKE_LIBRARY_PATH with no success, the only thing that worked was appending the C*Flags.